src/components/tests/AllGames.test.js

```diff
@@ -0,0 +1,19 @@
+import React from 'react';
+import renderer from 'react-test-renderer';
+import { BrowserRouter as Router } from 'react-router-dom';
+import { Provider } from 'react-redux';
+import store from '../../reduxFeatures/store';
+import AllGames from '../AllGames';
+
+describe('AllGames Test', () => {
+  it('renders correctly', () => {
+    const AllGamesTest = renderer.create(
+      <Provider store={store}>
```

Comment:
The project uses Redux for state management and data retrieved from the API are stored in the Redux store correctly 😉 Fantastic job!